src/Umbraco.Core/EmbeddedResources/Lang/tr.xml

```diff
@@ -654,14 +657,15 @@
     <key alias="first">İlk</key>
     <key alias="focalPoint">Odak noktası</key>
     <key alias="general">Genel</key>
+    <key alias="generic">Genel</key>
     <key alias="groups">Gruplar</key>
     <key alias="group">Grup</key>
     <key alias="height">Yükseklik</key>
     <key alias="help">Yardım</key>
     <key alias="hide">Gizle</key>
     <key alias="history">Geçmiş</key>
     <key alias="icon">Simge</key>
-    <key alias="id">Kimlik</key>
+    <key alias="id">Numara</key>
```

Comment:
```suggestion
    <key alias="id">Belirleyici Numara</key>
```
Bence id nin daha uygun bir tanimi?

Reply:
Bence de uygun. Bu ve diğer belirttiğin değişiklikleri yapıyorum şimdi. Bu arada Türkçe çeviri ihtiyacı olan başka alanlar da var, oralara da fırsat buldukça el atacağım. Teşekkürler.